### ISSUE TITLE
How to save the best training model?

### ISSUE THREAD

**Issue by gouzi1209** [sciann/sciann#52]
Dear professor, 

How to save the best training model? 
I tried to add the following code like Keras during training, but it got an error
('SciKernelInitializer' object has no attribute 'bias')

model_checkpoint = ModelCheckpoint(location, monitor='loss', save_best_only=True, mode='auto')
model.train([x1, x2], [y, ly], epochs=200, batch_size=64, callbacks=[model_checkpoint])

kind regards,
shuyi zhou

**Comment by ehsanhaghighat** [comment]
Hi Shuyi,

The keyword is save_weights and it automatically adds the right callback to the stack.

Best,
Ehsan

**Comment by gouzi1209** [comment]
Dear professor,
 
I made the following changes in the model.train, but the following error occurs.

(model.train([x1, x2], [y, ly], epochs=200, batch_size=64, save_weights={'best': True, 'path': '/home/zhousy/'}))

local variable 'model_check_point' referenced before assignment

I hope you can give me some guidance.

best regards,
shuyi zhou
